config/common/common.go

```diff
@@ -93,3 +103,102 @@ func GetFullyQualifiedIDFn(serviceProvider string, keyPairs ...string) tjconfig.
 		return path, nil
 	}
 }
+
+var referenceRules = [][]string{
```

Comment:
Added a comment to code:

`In the regular expressions used when determining the references, we can observe many rules that have common strings. Some of these may be more special than others. For example, "ex_subnet$" is a more special case than "subnet$". By putting the more specific rules before the general rules in array, processing and capturing the specific rules first will be possible. Since there is no fixed index for key-value pairs in maps, it is not possible to place rules from specific to general. Therefore, array is used here.`